### PR TITLE
Create post to wishlist database

### PR DESCRIPTION
**Pull request by aitzeng**

### CODE REVIEW

**Review by Dmigdol — APPROVED** [review]

**Review by pwang0407 — APPROVED** [review]
good job

File: server/database/schemas/blue-ocean-schema.sql [hunk 1]
```diff
@@ -48,11 +48,11 @@ CREATE TABLE "messages" (
 CREATE TABLE "wishlist" (
   "id" serial PRIMARY KEY,
   "user_id" text,
+  "album_id" text,
```
Comment (seguyy) [inline server/database/schemas/blue-ocean-schema.sql:50]: @aitzeng why did you add album_id back? I don't think we need that

File: server/index.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ app.use(express.json());
 app.use('/api/record-catalog', router.helpers);
 app.use('/api/trading-platform', router.tradingPlatform);
 app.use('/api/profile', router.profile);
+app.use('/wishlist', router.wishlist);
```
Comment (seguyy) [inline server/index.js:21]: let's make this also api/wishlist
